ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/SqlToJavaVisitor.java

```diff
@@ -1079,13 +1079,14 @@ public Pair<String, SqlType> visitSubscriptExpression(
 
         case MAP:
           final SqlMap map = (SqlMap) internalSchema;
+          final String baseCode = process(node.getBase(), context).getLeft();
```

Comment:
The issue is that calling `proccess` node twice is not idempotent on `FunctionCall` nodes.